exercises/concept/calculator-conundrum/.docs/instructions.md

```diff
@@ -0,0 +1,36 @@
+# Instructions
+
+In this exercise you will be building error handling for a simple integer calculator. To make matters simple, methods for calculating addition, multiplication and division are provided.
+
+The goal is to have a working calculator that returns a string with the following pattern: `16 + 51 = 67`, when provided with arguments `16`, `51` and `+`.
+
+<!---
+TODO
+-->
```

Comment:
We going to go update this once the Java code is ready? I think we are pretty much there wrt the Java code at this point.